src/ILLink.CodeFix/BaseAttributeCodeFixProvider.cs

```diff
@@ -108,7 +108,7 @@ internal enum AttributeableParentTargets
 
 		internal abstract SyntaxNode[] GetAttributeArguments (SemanticModel semanticModel, SyntaxNode targetNode, CSharpSyntaxNode declarationSyntax, SyntaxGenerator generator, Diagnostic diagnostic);
 
-		internal static bool HasPublicAccessibility (IMethodSymbol? m)
+		internal static bool HasPublicAccessibility (ISymbol? m)
 		{
```

Comment:
Should we add a null check here?

Reply:
A property check is an implicit `null` check (note, there are no nullability warnings from here-on).